packages/source-iotsitewise/src/completeDataStreams.ts

```diff
@@ -13,34 +16,93 @@ const toDataType = (propertyDataType: PropertyDataType | string | undefined): Da
   return 'NUMBER';
 };
 
+const parseAlarmData = ({ alarmSource, data }: { alarmSource: AlarmSource; data: DataPoint[] }): DataPoint[] => {
```

Comment:
should probably just move this type of code into it's own file in the `alarms` folder - we have sitewise alarm logic spread across the various layers, if there was some way to manage it all within the `alarms` folder that we can i think that may help simplify the other layers of the codebase

Reply:
Agreed.